<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>学生</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="3" rowspan="8">
         <widget class="QTableWidget" name="tbwStu">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leStuName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leStuBd">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>性别:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>奖学金:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="btnStuChg">
          <property name="minimumSize">
           <size>
            <width>195</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="btnStuIns">
          <property name="minimumSize">
           <size>
            <width>195</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="leStuDep">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>年龄:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>姓名:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leStuNum">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cbStuSch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>否</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>是</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>所在系:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>学号:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbStuSex">
          <property name="minimumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="course_tabWidget">
       <attribute name="title">
        <string>课程</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>课程号:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="crs_cno_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>课程名:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="crs_cname_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>现行课:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="crs_cpno_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>学分:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="crs_credit_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="crs_insert_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="crs_update_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="crs_del_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>457</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="grade_tabWidget">
       <attribute name="title">
        <string>成绩</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>学号:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="grd_sno_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>课程号:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="grd_cno_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>成绩:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="grd_grade_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="grd_insert_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="grd_update_pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>195</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>539</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="all_tabWidget">
       <attribute name="title">
        <string>综合</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>按系统计</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>系名:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="all_dept_lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>195</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="all_dept_sti_pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>195</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>统计</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="all_dept_ord_pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>195</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>排名</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>学生信息查询</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>学号:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="all_stu_lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>195</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="all_stu_pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>195</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>463</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
